python/desc/twinkles/io.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, print_function, division
```

Comment:
Would `twinkles_io` be ok? I agree that io.py is probably too generic, though it does not matter a lot, and I would be happy to change it to that or any other suggestions.